File: content/code/python/even_time_difference.py
import time
def check_time_to_now_even(seconds):
    """
    Checks whether time difference between now and given time in seconds is even
    """
    if seconds < 0:
        raise ValueError('received negative input')
   
    now=time.time()
    return (seconds-now)%2 == 0

File: content/code/R/even_time_difference.R
#' Check time to now even
#'
#' Checks whether time difference between now and given time in seconds is even
#'
#' @param seconds input time in whole seconds
#'
#' @return returns true or false
check_time_to_now_even <- function(seconds) {
  if(seconds < 0) stop('received negative input')
  now <- as.integer(format(Sys.time(), '%s'))
  return((seconds - now) %% 2 == 0)
}
